microlith/loki/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -273,3 +273,23 @@ groups:
           title: "Large Number of Dropped Ticks Detected (node: {{ $labels.hostname }})"
           description: Couchbase Server nodes regularly send heartbeat ticks to each other. Dropped ticks are usually a sign of resource contention.
           remediation: Increase number of CPUs available to Couchbase Server or, if you are running a virtualised environment, check for VM overcommitment.
+
+      - alert: CB90067-serviceRollback
+        expr: |
+          count_over_time( {file="memcached.log"} |= "requires rollback to seqno:0" != "eq_dcpq:replication" [1h]) > 0
+        labels:
+          job: couchbase_fluent_bit
+          kind: node
+          severity: warning
+          health_check_id: CB90067
+          health_check_name: serviceRollback
+          cluster: '{{ $labels.couchbase_cluster }}'
+          node: '{{ $labels.hostname }}'
+        annotations:
+          title: "Service DCP Rollback to Zero (node: {{ $labels.hostname }})"
+          description: A service has been forced to DCP rollback to zero.
+            A DCP rollback is when a client connects to the Data Service with newer mutations that are not present on the Data Service.
+            The client must rollback or undo some mutations to align with the mutations on the Data Service.
+            If the Data Service is rolled back to 0, it means the service is attempting to resynchronize the entirety of the data set.
```

Comment:
You don't need the full description in the alert rule itself, one or two sentences is enough - that's what the docs are for.

Reply:
I shortened it down